Server is running at http://localhost:3000
Server is running at http://localhost:3000
Server is running at http://localhost:3000
Error: Command failed: python3 algorithms/blowfish/decrypt.py "9955960FC85E5B2DF971E85342CC4A13" "1234"
Traceback (most recent call last):
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/blowfish/decrypt.py", line 39, in <module>
    print(decrypted_text.decode())
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xbd in position 3: invalid start byte

Server is running at http://localhost:3000
Error: Command failed: python3 algorithms/blowfish/decrypt.py "9955960FC85E5B2DF971E85342CC4A13" "1234"
Traceback (most recent call last):
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/blowfish/decrypt.py", line 39, in <module>
    print(decrypted_text.decode())
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xbd in position 3: invalid start byte

Server is running at http://localhost:3000
Server is running at http://localhost:3000
Error: Command failed: python3 algorithms/Grasshopper/decrypt.py "9iJvcrXmM4xnkBNNH9eT8pW3KCrbQ5ewUaUdBaHlTOOOg5HzILQTmcKKdJ92YRGc2XgQqeqn3K17N+cLsTl3/8g56vv6bAiX" "123%5512"
Traceback (most recent call last):
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/Grasshopper/decrypt.py", line 17, in <module>
    decrypted_text = decrypt(data, password)
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/Grasshopper/decrypt.py", line 6, in decrypt
    decrypted_text = gh.decrypt(text, K)
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/Grasshopper/main.py", line 214, in decrypt
    textDecrypted = S(L(X(textDecrypted, K[i]), 'reverse'), 'reverse')
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/Grasshopper/main.py", line 5, in xor_func
    input1AsInteger = int(input1AsString, in_code)
ValueError: invalid literal for int() with base 16: '9iJvcrXmM4xnkBNNH9eT8pW3KCrbQ5ew'

Error: Command failed: python3 algorithms/blowfish/decrypt.py "9iJvcrXmM4xnkBNNH9eT8pW3KCrbQ5ewUaUdBaHlTOOOg5HzILQTmcKKdJ92YRGc2XgQqeqn3K17N+cLsTl3/8g56vv6bAiX" "123%5512"
Traceback (most recent call last):
  File "/Users/sl4sh73r/Desktop/Deleted/кунин/Лаба реворк/backend/algorithms/blowfish/decrypt.py", line 39, in <module>
    print(decrypted_text.decode())
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf5 in position 0: invalid start byte

Server is running at http://localhost:3000
